frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BasePushBranchAction.java

```diff
@@ -47,7 +47,7 @@ public IEnhancedLambdaLogger log() {
   }
 
   @Override
-  public List<SyncToRemoteStatus.Relation> getEligibleRelations() {
+  public List<SyncToRemoteStatus> getEligibleRelations() {
```

Comment:
Rename the method adequately

Reply:
Should it be named `getEligibleStatuses` now?

Reply:
Yes (pls don't resolve a thread if you wait for a response coz the reviewers are then more likely to overlook)